python/audit/tests/test_config_log.py

```diff
@@ -32,47 +33,49 @@ def setUp(self) -> None:
         super().setUp()
 
         self.config = ObjectConfig.objects.create(current=1, previous=1)
-        bundle = Bundle.objects.create()
-        prototype = Prototype.objects.create(bundle=bundle)
-        cluster = Cluster.objects.create(prototype=prototype, config=self.config)
+        self.bundle = Bundle.objects.create()
+        prototype = Prototype.objects.create(bundle=self.bundle)
+        self.cluster = Cluster.objects.create(prototype=prototype, config=self.config)
         ConfigLog.objects.create(obj_ref=self.config, config="{}")
         self.group_config = GroupConfig.objects.create(
             name="test_group_config",
-            object_id=cluster.pk,
+            object_id=self.cluster.pk,
             object_type=ContentType.objects.get(app_label="cm", model="cluster"),
             config_id=self.config.pk,
         )
 
-    def check_log(self, res: Response, log: AuditLog) -> None:
-        assert log.audit_object.object_id == res.data["id"]
-        assert log.audit_object.object_name == str(ConfigLog.objects.get(pk=res.data["id"]))
-        assert log.audit_object.object_type == AuditObjectType.Cluster
-        assert not log.audit_object.is_deleted
-        assert log.operation_name == "Cluster config log updated"
-        assert log.operation_type == AuditLogOperationType.Update
-        assert log.operation_result == AuditLogOperationResult.Success
-        assert isinstance(log.operation_time, datetime)
-        assert log.user.pk == self.test_user.pk
-        assert isinstance(log.object_changes, dict)
-
-    def check_log_denied(self, log: AuditLog) -> None:
-        assert not log.audit_object
-        assert log.operation_name == "config log updated"
-        assert log.operation_type == AuditLogOperationType.Update
-        assert log.operation_result == AuditLogOperationResult.Denied
-        assert isinstance(log.operation_time, datetime)
-        assert log.user.pk == self.no_rights_user.pk
-        assert isinstance(log.object_changes, dict)
+    def check_log(
+        self,
+        log: AuditLog,
+        operation_name: str,
+        operation_result: AuditLogOperationResult,
+        user: User,
+    ) -> None:
+        self.assertEqual(log.audit_object.object_id, self.cluster.pk)
+        self.assertEqual(log.audit_object.object_name, self.cluster.name)
+        self.assertEqual(log.audit_object.object_type, AuditObjectType.Cluster)
+        self.assertFalse(log.audit_object.is_deleted)
+        self.assertEqual(log.operation_name, operation_name)
+        self.assertEqual(log.operation_type, AuditLogOperationType.Update)
+        self.assertEqual(log.operation_result, operation_result)
+        self.assertIsInstance(log.operation_time, datetime)
+        self.assertEqual(log.user.pk, user.pk)
+        self.assertIsInstance(log.object_changes, dict)
```

Comment:
I'm thinking of adding an explicit check for `self.assertEqual(log.object_changes, {})` so that in the future when we implement storage `object_changes`, tests don't mask the problem.